### ISSUE TITLE
EdgeHTML: support aria-role=alert

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#8466]
Hi,

Somewhat related to #8448: In Microsoft Edge, aria-role=alert is not supported yet.

### Steps to reproduce:

1. Visit the following site with multiple web browsers, including with Edge: https://www.w3.org/TR/wai-aria-practices/examples/alert/index.html
2. Click "trigger alert" button.

### Actual behavior:
No feedback from NVDA in Microsoft Edge.

### Expected behavior:
NVDA should announce alert text.

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version:
Next.15445

#### Windows version:
Windows 10 Version 1803

#### Name and version of other software in use when reproducing the issue:
Microsoft Edge

#### Other information about your system:
Multiple systems, all part of Windows Insider Program.

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
Yes - master, 2018.2.1

### Technical details

Alerts are fired as live region changed events. It was used to be system alert, but apparently this has changed (need to verify this). Probing the element with ariaProperties says role=alert, which should then allow NVDAObjects.UIA.edge.EdgeNode.event_liveRegionChanged handler to fetch alert texts and announce it. Note that the name might be an empty string, but child count might be greater than 1, in which case probing the children should do.

Thanks.

**Comment by josephsl** [comment]
CC @tgeczy, @MichaelDCurran

**Comment by josephsl** [comment]
Hi,

In consideration for people using older Windows 10 releases, I advise not closing this issue even when Chromium Edge stable version is released.

Thanks.

**Comment by josephsl** [comment]
Hi,

Actually, it would be best to close this for now as Chromium Edge is about to go live. We will revisit this problem if there are situations where this must be resolved.

Thanks.

**Comment by Stefany93** [comment]
Are there any plans for Edge to support `role="alert"`?

**Comment by josephsl** [comment]
Hi, at least for EdgeHTML Edge, not at the moment as it is deprecated. Chromium-based Edge 79 and later might be the place to add this functionality. Thanks.

**Comment by josephsl** [comment]
Hi,

2021 update: Microsoft noted that support for legacy Edge powered by EdgeHTML has ended as of March 9, 2021. Therefore, it is becoming unlikely that this issue will be reopened again.

Thanks.
